tests/conftest.py

```diff
@@ -31,7 +31,7 @@ def node(Ansible, Interface, Command, request):
     if node_type == "mgrs" and ceph_stable_release == "jewel":
         pytest.skip("mgr nodes can not be tested with ceph release jewel")
 
-    journal_collocation_test = ansible_vars.get("journal_collocation") or ansible_vars.get("dmcrypt_journal_collocation")
+    journal_collocation_test = ansible_vars.get("osd_scenario", 'collocated')
```

Comment:
This needs to return a boolean. I think what you'll need here is: ``journal_collocation_test = ansible_vars.get("osd_scenario") == "collocated"``